### PR TITLE
Feature/user permissions

### PR DESCRIPTION
**Pull request by martinyde**

### CODE REVIEW

File: config/install/user.role.site_admin.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
+langcode: da
```
[the remark this answers is not in the document]
Comment (perthaarhus) [reply config/install/user.role.site_admin.yml:1]: The site admin is requested to have access to the website basic functions without opening up for the big administrator.

[the remark this answers is not in the document]
Comment (martinyde) [reply config/install/user.role.site_admin.yml:1]: As described by @perthaarhus. There is a request to separate the Administrator role with unlimited access from a site administrator role used by the client/product owner.

File: config/install/user.role.forloeb_designer.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+langcode: da
```
[the remark this answers is not in the document]
Comment (martinyde) [reply config/install/user.role.forloeb_designer.yml:1]: The two roles have different access.
Flow designer has maestro access which forloebsdesigner /selvbetjeningsdesigner does not. This is a change request that was agreed upon.

File: config/install/user.role.site_admin.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (andriyun) [reply config/install/user.role.site_admin.yml:1]: Ok, I don't mind having two roles for administrative purposes :)
But I think If two roles have different purposes then should have a clear self-talkative name.

**Comment by andriyun** [comment]
@martinyde, @perthaarhus Changelog still not updated.
All the rest looks good to me andapproved.

**Review by andriyun — COMMENTED** [review]
@martinyde, @perthaarhus Changelog still not updated.
All the rest looks good to me and approved.

**Review by rimi-itk — APPROVED** [review]

**Pull request merged**